tests/integration_tests/oad_process/data/CeRAS01_legacy_breguet_result.xml

```diff
@@ -1,6 +1,6 @@
 <!--
-  ~ This file is part of FAST-OAD : A framework for rapid Overall Aircraft Design
-  ~ Copyright (C) 2020  ONERA & ISAE-SUPAERO
+  ~ This file is part of FAST-OAD_CS25
```

Comment:
FAST-OAD_CS25 ?

Reply:
Yep. To have our integration tests working with last version of FAST-OAD-CS25, I copied the xml from the CS25 repo. Strictly speaking, keeping the CS25 copyright is the right move, I think.
Anyway, given we own the 2 repos, this is of very little importance.